src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -94,15 +92,12 @@
     <Compile Remove="runtime\Math\Functions\Single\ILogB.cs" />
     <Compile Remove="runtime\Math\Functions\Single\Log2.cs" />
     <Compile Remove="runtime\Math\Functions\Single\ScaleB.cs" />
+    <Compile Remove="runtime\PacketTracer\**\*.cs" />
```

Comment:
Has this functionality been ported over from ARM?

Reply:
No, but with https://github.com/dotnet/performance/blob/58e6c75de7a4ceb2f84af30d17d839ec7ffbd300/src/benchmarks/micro/runtime/PacketTracer/Render.cs#L15 we are prohibiting it to be executed on ARM, as only x86 and x64 are allowed